packages/apm/src/integrations/tracing.ts

```diff
@@ -202,6 +211,16 @@ export class Tracing implements Integration {
       });
     }
 
+    if (Tracing.options.discardBackgroundSpans && global.document) {
+      document.addEventListener('visibilitychange', () => {
+        if (document.hidden && Tracing._activeTransaction) {
+          logger.log('[Tracing] Discarded active transaction incl. activities since tab moved to the background');
+          Tracing._activeTransaction = undefined;
+          Tracing._activities = {};
+        }
+      });
```

Comment:
@HazAT Should `startIdleTransaction()` be called whenever `document.hidden === true`? e.g. user focuses the tab.

Reply:
@dashed What would that be measuring? Normally our auto-created transactions are about pageload, but if the page is already loaded, and you just refocus the tab...